targets/TARGET_WIZNET/TARGET_W7500x/analogin_api.c

```diff
@@ -57,12 +55,8 @@ void analogin_init(analogin_t *obj, PinName pin)
     // Save pin number for the read function
     obj->pin = pin;
 
-    // The ADC initialization is done once
-    if (adc_inited == 0) {
-        adc_inited = 1;
-
-        ADC_Init();
-    }
+    // The ADC initialization
+    ADC_Init();
```

Comment:
I believe I asked earlier but cant find my comment. Why are you removing this static init variable? If I have 2 pins, this will reinit ADC 2x (power down and up as I can see in this changeset)? Is this desirable behavior? what was wrong with code as it was?